### ISSUE TITLE
Difference between "pretrain stage" and "coarse stage"

### ISSUE THREAD

**Issue by bennyguo** [Gorilla-Lab-SCUT/Fantasia3D#20]
Hi Rui, thanks for releasing the code! I noticed that there's an `if_pretrain` flag in `train.py` and the number of iterations for this "pretrain" stage is hard-coded to 400. What's the intention of this pretrain stage and how is it different from the coarse stage controlled by `FLAGS.coarse_iter`? Thank you!

**Comment by ARuiChen** [comment]
Hi, Yuanchen! Your project is very nice! 
"pretrain" stage initializes the DMtet to an ellipsoid or a custom mesh. I found that optimizing the parameters of the pre-training and formal training stages using the same optimizer can stabilize the training process.
"FLAGS.coarse_iter" represents the iterations of the early stage using concatenated normal and mask as the latent code.

**Comment by bennyguo** [comment]
Thank you! BTW, a few typos in `train.py`: `if_nomral` and `if_pretain` should be `if_normal` and `if_pretrain` :)
